Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -22,11 +22,12 @@ class TabScrollingController: NSObject {
     weak var tab: Tab? {
         willSet {
             self.scrollView?.delegate = nil
-            self.scrollView?.removeGestureRecognizer(panGesture)
+            self.scrollView?.panGestureRecognizer.removeTarget(self, action: nil)
         }
 
         didSet {
-            self.scrollView?.addGestureRecognizer(panGesture)
+            self.scrollView?.panGestureRecognizer.addTarget(self, action: #selector(handlePan))
+            self.scrollView?.panGestureRecognizer.maximumNumberOfTouches = 1
```

Comment:
Should we be modifying the scroll view's pan gesture? This may have side-effects

Reply:
Agreed. I copied what was done for the PanGesture created. Looking at it now there is no need to update this value.